core/src/test/java/io/questdb/test/cairo/fuzz/DedupInsertFuzzTest.java

```diff
@@ -980,9 +1023,14 @@ private void validateNoTimestampDuplicates(
             foundSymbols = new boolean[symbols.length];
         }
 
+        if (convertToParquet) {
+            // force reload parquet partitions
```

Comment:
This seems to be a hack. The reader should get reloaded when we acquire it from the pool. Why do we need this call in case of parquet partitions?

Reply:
Should be fixed in 35646d4d0dd97b8c827aad6a0e5590063d5ba0ec
@eugenels could you take a look?